bin/runtime/src/lib.rs

```diff
@@ -698,6 +703,12 @@ impl pallet_treasury::Config for Runtime {
     type SpendOrigin = frame_support::traits::NeverEnsureOrigin<u128>;
     type SpendPeriod = SpendPeriod;
     type WeightInfo = pallet_treasury::weights::SubstrateWeight<Runtime>;
+    type AssetKind = ();
+    type Beneficiary = Self::AccountId;
+    type BeneficiaryLookup = IdentityLookup<Self::AccountId>;
+    type Paymaster = PayFromAccount<Balances, TreasuryAccount>;
+    type BalanceConverter = UnityAssetBalanceConversion;
+    type PayoutPeriod = ConstU32<0>;
```

Comment:
I'm afraid I have no idea what this configuration does. At this point we don't "actively" use pallet_treasury, it's only accumulating some fees etc., but we don't do anything with them. Hopefully the new configuration is not doing anything crazy :P If nobody knows, then I guess this requires creating a jira task to research this, and make sure it does not affect us negatively. But I wouldn't consider this a blocker for merging.

Reply:
I'll create a task.